code/modules/antagonists/heretic/knowledge/rust_lore.dm

```diff
@@ -71,7 +71,16 @@
 
 /datum/heretic_knowledge/rust_fist/proc/on_mansus_grasp(mob/living/source, mob/living/target)
 	SIGNAL_HANDLER
-	if(source.a_intent == INTENT_HARM && !iscarbon(target))
+	var/static/list/always_hit_typecache = typecacheof(list(
+		/mob/living/carbon,
+		/mob/living/silicon,
+		/mob/living/simple_animal/bot,
+		/obj/item/storage/secure/safe/caps_spare,
```

Comment:
why this specifically?

Reply:
Don't think there will ever be a case that you don't want to hit the safe if you are next to it and clicking on it with your green hand.